src/t.ts

```diff
@@ -0,0 +1,1891 @@
+  },
+  UnionAcc,
+> = obj<
+  Type extends {
+    kind: 'INTERFACE' | 'UNION';
+    possibleTypes: any;
+  }
+    ? objValues<{
+        [PossibleType in Type['possibleTypes']]: getPossibleTypeSelectionRec<
+          Selections,
+          PossibleType,
+          Type,
+          Introspection,
+          Fragments,
+          UnionAcc
+        >;
+      }>
+    : Type extends {
+          kind: 'OBJECT';
+          name: any;
+        }
+      ? getPossibleTypeSelectionRec<
+          Selections,
+          Type['name'],
+          Type,
+          Introspection,
+          Fragments,
+          UnionAcc
+        >
+      : {} | UnionAcc
+>;
+
+type _getPossibleTypeSelectionRec<
+  Selections,
+  PossibleType extends string,
+  Type extends ObjectLikeType,
+  Introspection extends IntrospectionLikeType,
+  Fragments extends {
+    [name: string]: any;
+  },
+  UnionAcc,
+  IntersectionAcc,
+> = Selections extends [infer Node, ...infer Rest]
+  ? _getPossibleTypeSelectionRec<
+      Rest,
+      PossibleType,
+      Type,
+      Introspection,
+      Fragments,
+      UnionAcc,
+      | ((Node extends { kind: Kind.FRAGMENT_SPREAD | Kind.INLINE_FRAGMENT }
+          ? getSpreadSubtype<Node, Type, Introspection, Fragments> extends infer Subtype extends
+              ObjectLikeType
+            ? PossibleType extends getTypenameOfType<Subtype>
+              ? getFragmentSelection<
+                  Node,
+                  Subtype,
+                  Introspection,
+                  Fragments,
+                  isOptional<Node> extends true ? {} : never
+                >
+              : {}
+            : Node extends { kind: Kind.FRAGMENT_SPREAD; name: any }
+              ? makeUndefinedFragmentRef<Node['name']['value']>
+              : {}
+          : Node extends { kind: Kind.FIELD; name: any; selectionSet: any }
+            ? isOptional<Node> extends true
+              ? {
+                  [Prop in getFieldAlias<Node>]?: Node['name']['value'] extends '__typename'
+                    ? PossibleType
+                    : unwrapType$1<
+                        Type['fields'][Node['name']['value']]['type'],
+                        Node['selectionSet'],
+                        Introspection,
+                        Fragments,
+                        getTypeDirective<Node>
+                      >;
+                }
+              : {
+                  [Prop in getFieldAlias<Node>]: Node['name']['value'] extends '__typename'
+                    ? PossibleType
+                    : unwrapType$1<
+                        Type['fields'][Node['name']['value']]['type'],
+                        Node['selectionSet'],
+                        Introspection,
+                        Fragments,
+                        getTypeDirective<Node>
+                      >;
+                }
+            : {}) &
+          IntersectionAcc)
+      | UnionAcc
+    >
+  : IntersectionAcc | UnionAcc;
+
+type getPossibleTypeSelectionRec<
```

Comment:
This looks to be the build file can we remove this

Reply:
thanks, file removed